pkg/kubelet/apis/cri/v1alpha1/runtime/constants.go

```diff
@@ -25,3 +25,30 @@ const (
 	// NetworkReady means the runtime network is up and ready to accept containers which require network.
 	NetworkReady = "NetworkReady"
 )
+
+// LogStreamType is the type of the stream in CRI container log.
+type LogStreamType string
+
+const (
+	// Stdout is the stream type for stdout.
+	Stdout LogStreamType = "stdout"
+	// Stderr is the stream type for stderr.
+	Stderr LogStreamType = "stderr"
+)
+
+// LogTag is the tag of a log line in CRI container log.
+// Currently defined log tags:
+// * First tag: Partial/End - P/E.
+// We can define new tags in the future if needed. However, if we find out
+// that a lot more metadata need to be introduced in the future, we should
+// switch to json log format instead.
+type LogTag string
+
+const (
+	// Partial means the line is part of multiple lines.
+	Partial LogTag = "P"
+	// End means the line is a single line or the end of multiple lines.
+	End LogTag = "E"
```

Comment:
nit: more explicit name would be better, .e.g., `LogTagEnd`

Reply:
Done.

Reply:
nit: Should we change end -> full. partial and P --> partial line and full and F --> full line?

Reply:
SGTM